force-app/main/default/lwc/platformDevCertCalculator/platformDevCertCalculator.html

```diff
@@ -0,0 +1,85 @@
+<template>
+    <lightning-card  title="PD1 Calculator">
+        <!--Process-->
+        <lightning-button 
+            variant="brand" 
+            label="Calculate"
+            title="Calculate" 
+            slot="actions" 
+            onclick={calculateScore}
+        ></lightning-button>
+        <!--Input Information-->
+        <div class="slds-p-horizontal_small">
+            <lightning-input 
+                type="number" 
+                name="devFundamentals" 
+                label="Developer Fundamentals: 23%"
+                value={devFundamentalsScore}
+                onchange={handleChange} 
+            ></lightning-input>
+            <lightning-input 
+                type="number" 
+                name="processAuto" 
+                label="Process Automation: 30%" 
+                value={processAutomationScore}
+                onchange={handleChange} 
+            ></lightning-input>
+            <lightning-input 
+                type="number" 
+                name="userInterface" 
+                label="User Interface: 25%" 
+                value={userInterfaceScore}
+                onchange={handleChange} 
+            ></lightning-input>
+            <lightning-input 
+                type="number" 
+                name="testDebugDeploy" 
+                label="Testing, Debugging, and Deployment: 22%" 
+                value={testingScore}    
+                onchange={handleChange} 
+            ></lightning-input>
+        </div>
+        <!--Display Results-->
+        <div slot="footer">
+            <h1 class="slds-text-heading_large slds-text-title_bold slds-text-title_caps resultText">Certification Results</h1>
+            <p class="slds-text-heading_medium" slot="footer">Certification Score: <span class="slds-text-color_destructive">{certificationScore}%</span></p>
```

Comment:
Try using a template literal to generate this in your js code and utilize it here